README.md

```diff
@@ -20,6 +22,34 @@ Note that `node-gyp` must be installed. See the
 [node-gyp](https://github.com/nodejs/node-gyp) documentation for more
 information.
 
+### npm
+
+It is not recommended to install `hsd` from npm's repositories
+but it is still possible to install using `npm` using a remote
+`git` repository. To install a particular version, a git tag can
+be used. It is recommended to use the latest version.
+
+```
+$ npm install -g https://github.com/handshake-org/hsd.git#v2.1.3
```

Comment:
Hard-coding a version  number here means constant maintenance of the README. I'm not sure the exact syntax but it might be more convenient to use something like `semver^2.0.0`

Reply:
We could use whatever GitHub uses to represent refs in documentation.